app/routes/v2/clusters.js

```diff
@@ -223,7 +223,6 @@ const updateClusterResources = async (req, res, next) => {
           const checkResourceSelfLink = (checkResource.object.metadata && checkResource.object.metadata.annotations && checkResource.object.metadata.annotations.selfLink) ? checkResource.object.metadata.annotations.selfLink : (checkResource.object.metadata ? checkResource.object.metadata.selfLink : null);
           // If the checked resource update is dedupable and for the same resource, it shouldn't be kept.
           if( selfLink == checkResourceSelfLink && dedupUpdates.includes(checkResourceType) ) {
-            req.log.warn({ org_id: req.org._id, cluster_id: req.params.cluster_id, update_selfLink: selfLink, update_type: type }, 'Duplicate update in same payload, truncated' );
```

Comment:
Is this generating excess noise in the logs, or some other reason for removing?

Reply:
Excess noise, and not an indicator of a _problem_ (log was added originally to aid in verification of de-duplication)